summingbird-storm/src/main/scala/com/twitter/summingbird/storm/Edges.scala

```diff
@@ -0,0 +1,115 @@
+package com.twitter.summingbird.storm
+
+import com.twitter.bijection.{ Injection, Inversion }
+import com.twitter.summingbird.storm.builder.{ EdgeGrouping, OutputFormat, Topology }
+import scala.util.Try
+import java.util.{ ArrayList => JAList, List => JList }
+import StormTopologyBuilder._
+
+/**
+ * This companion object contains all `Edge`s types used by Summingbird's storm topology.
+ */
+private[storm] object Edges {
+  def shuffleItemToItem[T](
+    sourceId: Topology.EmittingId[Item[T]],
+    destId: Topology.ReceivingId[Item[T]],
+    withLocal: Boolean
+  ): Topology.Edge[Item[T], Item[T]] = Topology.Edge(
+    sourceId,
+    EdgeFormats.item[T],
+    EdgeGrouping.shuffle(withLocal),
+    identity,
+    destId
+  )
+
+  def shuffleKeyValueToItem[K, V](
+    sourceId: Topology.EmittingId[KeyValue[K, V]],
+    destId: Topology.ReceivingId[Item[(K, V)]],
+    withLocal: Boolean
+  ): Topology.Edge[KeyValue[K, V], Item[(K, V)]] = Topology.Edge(
+    sourceId,
```

Comment:
This was untested because we don't use `KeyValue` type for now (will be used by `leftJoin`). I removed from this change from the PR.